camptix.php

```diff
@@ -6605,6 +6605,7 @@ function form_checkout() {
 			$this->error_flags['invalid_payment_method'] = true;
 		}
 
+		do_action( 'camptix_checkout_start', $_POST['tix_attendee_info'], $this->order );
```

Comment:
Do we know that `$this->order` is always set at this point?

Reply:
yup, looks like it. this page comes directly after user has selected the tickets that they want to buy/claim